.travis.yml

```diff
@@ -1,18 +1,75 @@
-os:
-  - osx
-  - linux
+sudo: false
 language: java
+dist: xenial
+
+stages:
+  - name: checks
+    if: tag IS present
+  - test
+  - name: deploy
+    if: tag IS present
+
+jobs:
+  fast_finish: true
+  include:
+
+    - stage: checks
+      name: "Check version"
+      addons:
+      before_install:
+        - echo "Skip before_install"
```

Comment:
Why is this step needed? Same as for `install`

Reply:
I need to invalidate the _global_ `before_install` (same for `install`), I don't want to spend time installing things that I'm not going to use in this stage, but I've just realized that there is a nicer approach that might work:

```
      before_install: true
      install: true
```